src/components/02-components/card-expreciences/card-experiences.tsx

```diff
@@ -0,0 +1,230 @@
+import { Button, Box, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CardNumber from '../card-number/card-number';
+
+export type BrandProps = {
+  heading?: string;
+  state: string;
+  description?: string;
+  program?: string;
+  dateStart: string;
+  dateEnd: string;
+  location?: string;
+  cta: string;
+  ctaTitle: string;
+  hours: string;
+  hoursText: string;
+  hoursCtaTitle: string;
+  hoursCtaUrl: string;
+  variation: number;
+  cardNumberVariation: string;
+};
+
+export default function CardExperiences ({ 
+  heading, 
+  state,
+  description, 
+  program, 
+  dateStart,
+  dateEnd,
+  location, 
+  ctaTitle,
+  cta, 
+  hours, 
+  hoursText, 
+  hoursCtaTitle, 
+  hoursCtaUrl,
+  variation,
+  cardNumberVariation, 
+}: BrandProps) {
+
+  const theme = useTheme();
+
+  // Variables according to the variant of the number of items
+  let position;
+  let itemsAligmend;
+  let stateMargin;
+  let stateMarginX;
+  let infoMargin;
+  let bodyWidth;
```

Comment:
This can be re-written like this:

```
const position = variation >= 2 ? 'column' : 'row';
const itemsAlignment = variation >= 2 ? 'flex-start' : 'center';
...
```

In that way you simplify the code and avoid using regular variables (constants are better).